deploy/inject.yaml.template

```diff
@@ -72,8 +72,8 @@ spec:
           imagePullPolicy: Always
           command:
             - ./appmeshinject
-            - -sidecar-image=${SIDECAR_IMAGE:-840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy:v1.11.2.0-prod}
-            - -init-image=${INIT_IMAGE:-111345817488.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager:v2}
+            - -sidecar-image=${SIDECAR_IMAGE:-840364872350.dkr.ecr.${AWS_REGION}.amazonaws.com/aws-appmesh-envoy:v1.11.2.0-prod}
```

Comment:
Is the injector available in all the same regions as aws-appmesh-envoy @nckturner ? Ideally this should work. But there is also the `MESH_REGION` variable. Shouldn't we just use that @jefp ?

Reply:
Hi @lavignes, yes, we could use the `MESH_REGION` variable.
The envoy, controller, proxy-route, and injector should be in the same region where the mesh is located. I'll push the update.